### PR TITLE
Issue #812: Correct Linting error

### PR DESCRIPTION
**Pull request by relentless-pursuit**
Resolves: #812

### CODE REVIEW

**Comment by romani** [comment]
Please remove extra unrelated commit

**Comment by relentless-pursuit** [comment]
> Please remove extra unrelated commit

Sure, would do that once i have green CI

File: checkstyle-tester/README_MANUAL_EXECUTION.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ external files and branches first.
 Anything that starts with `#` is considered a comment and is ignored.
 `projects-to-test-on.properties` is expected to be in the following format:
 
-> REPO_NAME|[local|git|hg]|URL|[COMMIT_ID]|[EXCLUDE FOLDERS]
+```properties
+REPO_NAME|[local|git|hg]|URL|[COMMIT_ID]|[EXCLUDE FOLDERS]|
+```
```
Comment (romani) [inline checkstyle-tester/README_MANUAL_EXECUTION.md:16]: Please do
```

`REPO_NAME|[local|git|hg]|URL|[COMMIT_ID]|[EXCLUDE FOLDERS]|`

```

Comment (relentless-pursuit) [reply checkstyle-tester/README_MANUAL_EXECUTION.md:16]: i see missing pipes or extra pipes. even if it doesn't lead to any ci issue, but let me know if it is an issue and needs correction. 
```
```properties
my_custom_project|local|/home/username/java/my_custom_project||
my_custom_checkstyle|git|/home/username/java/git-repos/checkstyle/checkstyle|master|**/ignoreFolderOrFiles/**/*
my_custom_repo|hg|/home/username/java/hg-repos/myRepo|default|
```
```

Comment (romani) [reply checkstyle-tester/README_MANUAL_EXECUTION.md:16]: It should be single line and naturally wrapped by browser

Comment (relentless-pursuit) [reply checkstyle-tester/README_MANUAL_EXECUTION.md]: image
this is how it looks in broswer

**Pull request merged**